README.md

```diff
@@ -64,22 +64,31 @@ export const defaultConfig: Config = {
 };
 ```
 
-See [config.ts](src/config.ts) for all available options. Here is a sample of the common configu options:
+See [config.ts](src/config.ts) for all available options. Here is a sample of the common config options:
 
 ```ts
 type Config = {
-  /** URL to start the crawl, if sitemap is provided then it will be used instead and download all pages in the sitemap */
+  
+  /** Required - URL to start the crawl, if sitemap is provided then it will be used instead and download all pages in the sitemap */
```

Comment:
What do you think of using tags instead, for example:

```suggestion
  /** 
  * URL to start the crawl, if sitemap is provided then it will be used instead and download all pages in the sitemap 
  * @required
  */
```

This way we could eventually bring jsdoc/typedoc into the mix to generate meaningful documentation from them. Just a suggestion, as even jsdoc/typedoc can infer whether a property is required or not based on its typings 🤗